src/test/scala/PSO/ParticleSpec.scala

```diff
@@ -46,17 +46,88 @@ class ParticleSpec
     objectiveFunction = obj,
     initialSolutions = solutions,
     initialVelocity = Some(List(5.0, 5.0)),
-    topology = ParticleTopology.FullyConnected
+    topology = ParticleTopology.FullyConnected,
+    terminationCriteria = TerminationCriteria(maxIterations = 1)
   )
   val psoSystemFactory = new PSOSystemFactory(conf)
   val pso = psoSystemFactory.build(randomSeed = Some(0))
   val particles = pso.particles
 
-  override def afterAll {
+
+  val masterRef: TestActorRef[SwarmMaster] = TestActorRef(
+    Props(new SwarmMaster(
+      X => SampleFunctions.sphere(X.asInstanceOf[List[Double]]),
+      ProblemType.Minimization,
+      TerminationCriteria()
+    ))
+  )
+  val master = masterRef.underlyingActor
+  val particleRef: TestActorRef[Particle] = TestActorRef(Props(new Particle(
+    List(1.0, 1.0),
```

Comment:
Might be useful to name each parameter for clarifty/ease of reading. 

acceleration = 1.4944, 
sampleSolution = ....
etc